robot/impl/local_robot_test.go

```diff
@@ -3154,7 +3154,7 @@ func TestModularResourceReconfigurationCount(t *testing.T) {
 
 	testutils.WaitForAssertion(t, func(tb testing.TB) {
 		tb.Helper()
-		test.That(tb, logs.FilterMessageSnippet("Module successfully restarted").Len(), test.ShouldEqual, 1)
+		test.That(tb, logs.FilterMessageSnippet("Module resources successfully re-added after module restart").Len(), test.ShouldEqual, 1)
```

Comment:
Yep! This should fix the issue. In a previous version of the RDK, `Module successfully restarted` meant that all resources were _also_ re-added to the module, but this is no longer the case (we rely on the resource manager handling the re-addition in the case of manual reconfigurations). Good find 🧑‍🔧 . I'm fine merging this now if @cheukt is, too.